### PR TITLE
Change all wallet language names into native names (and scripts)

### PR DESCRIPTION
**Pull request by NanoAkron**
Also changed 'Old English' to 'English Old', because 'Old English' is actually a language in its own right, and someone may want to make a dictionary for it at some point in the future. I don't know who, but let's leave the chance open ;)

### CODE REVIEW

**Comment by NanoAkron** [comment]
Needs a matching change in Monero-core which failed to rebase for some reason. Will get to it later today.

File: src/mnemonics/german.h [hunk 1]
```diff
@@ -1678,11 +1678,11 @@ namespace Language
         "Zündung",
         "Zweck",
         "Zyklop"
-      }), 4)
-    {
+      }), 4)
+    {
```
Comment (moneromooo-monero) [inline src/mnemonics/german.h]: That looks suspect... looks like a noop, also found in several places, and random #endif changes...

Comment (NanoAkron) [inline src/mnemonics/german.h]: I saw those. I didn't change a thing though - do you know how I'd remove these from the commit?

Comment (moneromooo-monero) [inline src/mnemonics/german.h]: An easy way is:
git reset HEAD^ (this "uncommits" the top commit)
git add -p src/mnemonics/german.h
Select which of the hunks you want to commit or not
Do this with the rest of the files that changed
git commit (or git commit -S if you sign your commits)

Comment (NanoAkron) [inline src/mnemonics/german.h]: Thanks. If it looks like it worked, I'll re-push, otherwise I'll leave as-is.

Comment (NanoAkron) [inline src/mnemonics/german.h]: Meh:
```
@@ -1678,11 +1678,11 @@ namespace Language
         "Zündung",
         "Zweck",
         "Zyklop"
-      }), 4)
-    {
+      }), 4)
+    {
       populate_maps();
     }
   };
 }
 
-#endif
+#endif
Stage this hunk [y,n,q,a,d,/,K,g,s,e,?]? n
<stdin>:10: trailing whitespace.
    German(): Base("Deutsch", std::vector<std::string>({
warning: 1 line adds whitespace errors.
```

Comment (moneromooo-monero) [inline src/mnemonics/german.h]: Alright. Though... the above looks ok I think.

**Pull request merged**